app/Http/Controllers/Adm/Mship/Account.php

```diff
@@ -325,6 +326,17 @@ public function postBanAdd(CreateRequest $request, AccountData $account)
             ->withSuccess('You have successfully banned this member.');
     }
 
+    public function getBans(AccountData $account)
+    {
+        $bans = BanData::isLocal()
+            ->orderByDesc('created_at')
+            ->paginate(15);
+
+        return $this->viewMake('adm.mship.account.ban.index')
+            ->with('bans', $bans)
+            ->with('account', $account);
```

Comment:
You're adding `$account` here but seem to be using the existing `$_account` variable in the view. This line can probably be removed?